frontend/javascripts/oxalis/view/right-border-tabs/connectome_tab/connectome_view.tsx

```diff
@@ -166,6 +166,9 @@ const synapseNodeCreator = (synapseId: number, synapsePosition: Vector3): Mutabl
   timestamp: Date.now(),
   bitDepth: 8,
   interpolation: false,
+  // Don't assume any additionalCoordinates here, because no 4D connectomes are
```

Comment:
Should we collect which features are not prepared for >3d somewhere?

Reply:
I also added this to https://github.com/scalableminds/webknossos/issues/7229.